### ISSUE TITLE
Error handler in PhoneGap Accelerometer Demos may ref non-existent fields

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#56]
```
What steps will reproduce the problem?
1.See Accelerometer js in PhoneGap demo
2.Error handler ref's 'code' and 'message' fields in error object
3.See PhoneGap doc on Accelerometer or PhoneGap java code - do these fields 
exist? Or what error info if any is there to report on error?


Fix error handler to give meaningful error data and/or remove refs to fields.

```

Original issue reported on code.google.com by `libby.ba...@gmail.com` on 21 Dec 2011 at 5:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 6:59
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `libby.ba...@gmail.com` on 7 Feb 2012 at 4:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
